sqlserver-delta-plugins/src/e2e-test/java/io.cdap.plugin/mssql/actions/MssqlActions.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.mssql.actions;
+
+import io.cdap.e2e.utils.AssertionHelper;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.e2e.utils.WaitHelper;
+import io.cdap.plugin.MssqlClient;
+import io.cdap.plugin.mssql.locators.Locators;
+import org.junit.Assert;
+
+import java.sql.SQLException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Replication Mssql Actions.
+ */
+public class MssqlActions {
+    public static String tableName = PluginPropertyUtils.pluginProp("mssqlSourceTable");
+    public static String schemaName = PluginPropertyUtils.pluginProp("mssqlSchema");
+    public static String datatypeValues = PluginPropertyUtils.pluginProp("mssqlDatatypeForInsertOperation");
+    public static String deleteCondition = PluginPropertyUtils.pluginProp("mssqlDeleteRowCondition");
+    public static String updateCondition = PluginPropertyUtils.pluginProp("mssqlUpdateRowCondition");
+    public static String updatedValue = PluginPropertyUtils.pluginProp("mssqlUpdatedRow");
```

Comment:
You are using these 3 variables only in ```executeCdcEventsOnSourceTable``` you can create them locally inside this method.

Reply:
Change Done.